comunas_geopandas.py

```diff
@@ -0,0 +1,74 @@
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry import Point, Polygon
+
+print('start to load csv')
+#load tele data
+teledata_location = 'H:\\OneDrive - Instituto Tecnologico y de Estudios Superiores de Monterrey\\MachineLearning_Extern\\ClassContent\\Reto\\Matrices de Viaje para Santiago Chile\\DataSet\\'
+df = pd.read_csv(teledata_location + '20210101_RM.csv')  
+print('finish loading tele data')
+
+
+#load map data
+map_loc = 'H:\\OneDrive - Instituto Tecnologico y de Estudios Superiores de Monterrey\\MachineLearning_Extern\\ClassContent\\Reto\\Actividades\\R13\\'
```

Comment:
Refer to `comunas_geopandas.py` file in `main` to update the paths. Local paths prevent other members of the team from running your script.

Create the required paths if you do not have them in your machine.